package/Maps/Yuri's Revenge/2_alpine_assault.map

```diff
@@ -0,0 +1,3127 @@
+; Map created with FinalAlert 2(tm) Mission Editor
+; Get it at http://www.westwood.com
+; note that all comments were truncated
+
+; This FA2 uses FA2sp created by secsome
+; Get the lastest dll at https://github.com/secsome/FA2sp
+; Current version : 1.6.3.1
+
+[Preview]
+Size=0,0,106,61
+
+[PreviewPack]
+1=BIACcgAEwBtAMnRABAAaQCSANMAVQASAAnIABMAbQDJ0QAQAGkAkgDTAFUAEgAJyAATAG0
+2=yAsAIAXQ5PDQ5PDQ6JQATAEE6PDQ4PDI4JgBTAFEAkgAJyAATAG0AydEAEABpAJIA0wBVA
```

Comment:
the preview pack looks pretty small?

Reply:
There's no preview on this map file (I selected no preview in Final Alert when saving to keep the file size small). I figured CNCNet adds a preview regardless.  Should I add the preview that Final Alert does automatically?

Reply:
yeah include the default preview, since that is what is shown in the loading screen, else it's black